### PR TITLE
[G2M] Range for monthly and annual digest, with API usage optimizations

### PR DESCRIPTION
**Pull request by woozyking**
motivated by the need to generate a raw summary for the year for [SR&ED prep work](https://eqworks.slack.com/archives/GH98ASPC4/p1611084153000100)

To make API calls more economical, for digests, PR enrichments (reviews/commits) and general comments enrichment are disabled.

`util.isPR()` is revised to work more reliably in the event of not performing PR enrichments.

a new `npm run range` command is added with 2 positional arguments (`npm run range <YYYY-MM-DD> <scope>`), for instance:

`npm run range` is equivalent to _this month_ (from start to end)
default

`npm run range 2020-12-01` is equivalent to the month of 2020-12 (December) month

`npm run range 2020-07-01 year` is equivalent to the year of 2020, as seen from ^

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,11 +23,11 @@
     "node": "14.x"
   },
   "dependencies": {
-    "@octokit/core": "^3.2.1",
-    "@slack/web-api": "^5.14.0",
+    "@octokit/core": "^3.2.4",
```
Comment (tlowande) [inline package.json:26]: out of curiosity: does it matter if [core](https://www.npmjs.com/package/@octokit/core) or [`@octokit/request`](https://www.npmjs.com/package/@octokit/request) for this project? the size difference doesn't seem relevant really.

File: daily.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const dailyPreviously = () => {
     issuesByRange({ start, end })
       .then((issues) => issues.filter(ignoreProjects))
       .then((issues) => issues.filter(ignoreBotUsers))
-      .then((issues) => enrichIssues({ issues, start, end })),
+      .then((issues) => enrichIssues({ issues, start, end, skipEnrichPRs: false, skipEnrichComments: false })),
```
Comment (tlowande) [inline daily.js:24]: don't you have to do the same on the `weekly.js` one since the default is always true?

**Review by tlowande — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (woozyking) [reply package.json:26]: yea not really as much. just wanted to pick the ones in need but may very well be the full request package (also i love their API design, very HTTP natural/intuitive)

File: daily.js (hunk 2 above)
Comment (woozyking) [reply daily.js:24]: ye default is true and digests (like weekly) don't really need them, aside from the previous dumb decision to format whether it's PR/Issues based on `enriched_commits` existence which has been simplified along in this PR.

**Review by hyx131 — APPROVED** [review]

**Pull request merged**